src/library_syscall.js

```diff
@@ -153,6 +153,19 @@ var SyscallsLibrary = {
       }
       return ret;
     },
+#else
+    // ASMFS filesystem disabled lite handling of stdout and stderr:
```

Comment:
this isn't ASMFS-specific, is it? I think this should say `FILESYSTEM=0`

Reply:
Err sorry, yeah, here I meant `MEMFS` and not `ASMFS`.